phpunit.xml.dist

```diff
@@ -71,8 +64,6 @@
         <testsuite name="recipe-blog">
             <directory>vendor/silverstripe/blog/tests</directory>
             <directory>vendor/silverstripe/lumberjack/tests</directory>
-            <directory>vendor/silverstripe/widgets/tests</directory>
-            <directory>vendor/silverstripe/content-widget/tests</directory>
             <directory>vendor/silverstripe/spamprotection/tests</directory>
             <directory>vendor/silverstripe/comments/tests</directory>
             <directory>vendor/silverstripe/comment-notifications/tests</directory>
```

Comment:
Shouldn't the two comment test suite be removed as well?

Reply:
yes